ui/panels-plugin/src/plugins/trace-table/DataTable.tsx

```diff
@@ -70,21 +73,34 @@ export function DataTable({ result }: DataTableProps) {
   );
 }
 
-function buildRow(theme: PersesChartsTheme, trace: TraceSearchResult): ReactNode {
+function buildRow(trace: TraceSearchResult): ReactNode {
   let totalSpanCount = 0;
   let totalErrorCount = 0;
   for (const stats of Object.values(trace.serviceStats)) {
     totalSpanCount += stats.spanCount;
     totalErrorCount += stats.errorCount ?? 0;
   }
 
+  const traceLinkParams = new URLSearchParams({
+    explorer: '1',
+    queries: `[{"kind":"TraceQuery","spec":{"plugin":{"kind":"TempoTraceQuery","spec":{"query":"${trace.traceId}"}}}}]`,
```

Comment:
This seems hacky, is there a better way to do it?
I could use `onClick`/`setQueries` and handle it client side, but then I'll loose the browser features of real links, like open a link in a new (background) tab, etc.

Reply:
Hmm, I don't think of a better solution :/

However, I think we should remove this magic number (1) and put it behind a variable 😄

Reply:
In which package should I define this constant? It belongs to the `explore` package, however we don't have a dependency from `panels-plugin` on the `explore` package (I'm not sure if I should add one? Doesn't seem right).

Maybe we can change `explorer=1` to `explorer=traces`, to avoid having a magic number in the code, and not need a dependency from `panels-plugin` to the `explore` package?